site/pages/pycon-thailand.th.rst

```diff
@@ -91,6 +91,39 @@ meet our keynote speaker
        และเขียนหนังสือชื่อ Python for Data Analysis และยังเป็นผู้สนับสนุน Arrow, Kudu (ช่วงเริ่มต้น)
        และโปรเจค Parquet ภายใต้มูลนิธิ Apache Software
 
+.. container:: jumbotron
+
+    .. class:: img-circle img-responsive col-md-4
+
+              .. image:: /tiwa-york.jpeg
+                     :alt: Tiwa York (portrait)
+                     :align: left
+                     :width: 200px
+
+    .. class:: col-md-8
+
+       .. raw:: html
+
+            <h1>ทิวา ยอรค</h1>
+
+       https://www.linkedin.com/in/tiwayork
+
+       ประธานเจาหนาที่บริหาร/เฮดโคช - Kaidee
+       
+       ทิวา ยอรค เฮดโคชแหงทีม Kaidee แพลตฟอรมที่เขาสรางขึ้นมาดวยความเชื่อที่วา “คนไทยทุกคนควรมีแพลตฟอรมสําหรับการซื้อขาย
+       ของออนไลนที่สามารถเขาถึงไดงาย” เขามุงมั่นที่จะพัฒนาตลาดออนไลนแหงนี้ควบคูไปกับการสรางและพัฒนาทีมเทคคนไทยที่มีความ
+       สามารถไมแพทีมเทคระดับโลก
+       
+       Kaidee เริ่มตนขึ้นในป 2011 จากทิวาและทีมอีก 5 คนที่รวมกันสรางแพลตฟอรมนี้ขึ้นมา จนทุกวันนี้กลายเปนที่รูจักและใชงานอยาง
```

Comment:
This looks weird here in GitHub, does it look ok on the actual site?

Reply:
Do you mean the blocky spaces?
The rendered version looks fine to me: https://deploy-preview-118--serene-mestorf-5c91d0.netlify.com/
I'll move the bulk of the bio to another page though.